### PR TITLE
Chapter 8 (cleanup and help pages)

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR creates help pages for chapter 8

### CODE REVIEW

File: content/resources/chapter-8/building-blocks.tsx [hunk 1]
```diff
@@ -213,6 +251,58 @@ const pythonChallengeFour = {
...
+  defaultCode: `const validateBlock = ()=> {
+  let fee = 0
+  let subsidy = getSubsidy(block["height"])
+
+  for (const tx of block["height"]) {
+    fee+= getTxFee(tx)
+  }
+  return subsidy + fee == block["txs"][0]["outputs"][0]["value"]
+  }
+}
```
Comment (benalleng) [inline content/resources/chapter-8/building-blocks.tsx:269]: I am not able to proceed using this code

Comment (benalleng) [reply content/resources/chapter-8/building-blocks.tsx:269]: lets use something like this.
I had trouble when trying to use the `block["height"]` object pointer style
``` suggestion
  defaultCode: `function validateBlock(block) {
  let subsidy = getSubsidy(block["height"]);
  let fee = 0;

  for (let i = 1; i < block["txs"].length; i++) {
    fee += getTxFee(block["txs"][i]);
  }
  
  return subsidy + fee === block["txs"][0]["outputs"][0]["value"]
}
```

**Review by benalleng — CHANGES_REQUESTED** [review]

File: content/resources/chapter-8/building-blocks.tsx [hunk 2]
```diff
@@ -165,7 +203,7 @@ const javascriptChallengeFour = {
     name: 'verify',
     args: [],
   },
-  defaultCode: `function get_subsidy(height) {
+  defaultCode: `const getSubsidy = (height)=> {
```
Comment (benalleng) [inline content/resources/chapter-8/building-blocks.tsx:206]: We should probably change this to your answer

File: content/resources/chapter-8/building-blocks.tsx [hunk 3]
```diff
@@ -213,6 +251,58 @@ const pythonChallengeFour = {
...
+  ],
+}
+
+const pythonChallengeFive = {
+  program: `print("KILL")`,
+  defaultFunction: {
+    name: 'verify',
+    args: [],
+  },
+  defaultCode: `def validate_block:
```
Comment (benalleng) [inline content/resources/chapter-8/building-blocks.tsx:288]: need `def validate_block(block):`
and `fee = 0`

File: content/resources/chapter-8/building-blocks.tsx (hunk 2 above)
Comment (0tuedon) [reply content/resources/chapter-8/building-blocks.tsx:206]: The big int?

**Review by benalleng — APPROVED** [review]
Any changes we can make in a future PR, this is a good start!

**Pull request merged**